admin/SWC_cover_letter.txt

```diff
@@ -0,0 +1,21 @@
+Dear SCF Mentoring Subcommittee,
+
+We are writing to express our interested to attend the two-day training course in December 2015 as one group composed of:
+
+Madeleine Bonsma (@mbonsma on GitHub),
+Lindsay Coome (@lcoome on GitHub),
+Lina Tran (@linamnt on Github),
+Sarah Meister (@thesarahmeister on GitHub),
+Elliott Sales de Andrade (@QuLogic on GitHub),
+Ricardo Harripaul (@ricardoharripaul on GitHub), and
+Derek Howard (@derekhoward on Github).
+
+Madeleine Bonsma is the organizer of a scientific progamming peer learning group at the University of Toronto (University of Toronto Scientific Coders - uoftcoders.github.io) and is on the waitlist for instructor training. Lindsay Coome is a session leader in the University of Toronto Scientific Coders group and attended a Software Carpentry workshop at SickKids Hospital in 2015. She is also on the waitlist for instructor training. Lina Tran is an organizer and member of the University of Toronto Scientific Coders group and attended a Software Carpentry workshop at SickKids Hospital in 2015. Sarah Meister is a session leader in the Scientific Coders group. Elliott Sales de Andrade is an organizer and member of the University of Toronto Scientific Coders group. Ricardo Harripaul is a session leader in the Scientific Coders group. Derek Howard is a member of the University of Toronto Scientific Coders group and previously went through SCW training but never completed his last pull request to improve the Python lesson.
+
+The University of Toronto Scientific Coders group will provide ongoing support and practice opportunities for SWC participants after-the-fact, ensuring that concepts learned can be effectively retained. 
+
+If we are selected for the instructor training we will use a seminar room in the Ontario Institute for Studies in Education (OI 2198 on Dec 7 and OI 8201 on Dec 8). We have confirmed reservations for these rooms. 
+
+We are planing to organize two workshops at the beginning of 2016. One will be Python based and will have NAME_OF_1ST_STUDENT, NAME_OF_2ND_STUDENT, NAME_OF_4TH_STUDENT and NSTRUCTOR_NAME as instructors. The other one will be R based and will have NAME_OF_3RD_STUDENT, NAME_OF_5TH_STUDENT, NAME_OF_6TH_STUDENT and INSTRUCTOR_NAME as instructors. We will use the same room used for the last workshop and we already talked with INSTRUCTOR_NAME about those workshops. 
```

Comment:
We will use the same room as used by UofT AIS for the last workshop (if you want!) Also, I think the python-based workshop would be a better fit for me.